tests/testing_data/cts/getCapabilities.xml

```diff
@@ -224,7 +224,7 @@
                     <ti:edition workUrn="urn:cts:latinLit:phi1276.phi001" urn="urn:cts:latinLit:phi1276.phi001.perseus-lat2">
                         <ti:label xml:lang="eng">Satires</ti:label>
                         <ti:description xml:lang="eng">
-            Juvenal and Persius: With An English Translation, G. G. Ramsay, 1918, William Heinemann; G. P. Putnam's Son, London
+            Juvenal and Persius: With An English XmlCtsTranslationMetadata, G. G. Ramsay, 1918, William Heinemann; G. P. Putnam's Son, London
```

Comment:
I am pretty sure you didn't want to change this. It does make for some funny metadata, however.